assignment_6.py

```diff
@@ -0,0 +1,59 @@
+# Rearranging cars
+
+def rearrange_cars(cars,order):
+    '''
+        Input: - 2 list of integers
+                cars =  a list of integers that represents the cars
+                order = a list of integers that represents in which order should the cars be rearranged
+        Returns a list that contains all the moves made to get the rearrangement        
+                
+    '''
+    empty = find_zero(cars) 
+    moves = []
+    dic = build_dic(order)
+    for i in range(len(cars)):
+        if cars[i] != order[i] and cars[i]!= 0:
+            car_pos = dic[cars[i]]      
+            
+            if car_pos != empty:
+                moves.append('move from {} to {}'.format(empty, car_pos))
+                
+            cars[empty], cars[car_pos] = cars[car_pos], cars[empty]
+            empty = car_pos
+            moves.append('move from {} to {}'.format(empty, i)) 
+    
+            cars[i], cars[empty] = cars[empty], cars[i]
+            empty = i
+
+    return moves   

```

Comment:
If I say
`return cars`
instead of
`return moves`

Should not we get the target arrangement?
If yes, then this is the output:
init: [6, 10, 2, 3, 8, 4, 5, 7, 1, 0, 9]
desired: [2, 1, 10, 8, 7, 3, 6, 9, 0, 4, 5]
actual: [2, 1, 10, 8, 7, 3, 6, 4, 9, 0, 5]

Please excuse me if I'm mistaken